test Fixture with speci@l chars/TestDeployInSpecificNamespace.java

```diff
@@ -0,0 +1,17 @@
+// camel-k: language=java
+
+import org.apache.camel.builder.RouteBuilder;
+
+public class TestDeployInSpecificNamespace extends RouteBuilder {
+  @Override
+  public void configure() throws Exception {
+
+      // Write your routes here, for example:
+      from("timer:java?period=1000")
+        .routeId("java")
+        .setBody()
+          .simple("Hello Camel K from ${routeId}")
+        .to("log:info");
+
+  }
+}
```

Comment:
I think this file is unrealted to this change

Reply:
Wops my bad. I'll remove it